roxctl/central/db/generate/volumes.go

```diff
@@ -0,0 +1,67 @@
+package generate
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"github.com/stackrox/rox/pkg/renderer"
+	"github.com/stackrox/rox/roxctl/common/environment"
+)
+
+func volumeCommand(name string) *cobra.Command {
+	return &cobra.Command{
+		Use:     name,
+		Short:   fmt.Sprintf("adds a %s", name),
+		Long:    fmt.Sprintf(`adds a %s external volume to Central DB`, name),
+		Example: "Enter Central volume type",
+	}
+}
+
+func externalVolume(cliEnvironment environment.Environment) *cobra.Command {
+	external := &renderer.ExternalPersistence{
+		DB: &renderer.ExternalPersistenceInstance{},
+	}
+	c := volumeCommand("pvc")
+	c.RunE = func(c *cobra.Command, args []string) error {
+		cfg.External = external
+		if err := validateConfig(&cfg); err != nil {
+			return err
+		}
+		return OutputZip(cliEnvironment.Logger(), cfg)
+	}
+	c.Flags().StringVarP(&external.DB.Name, "name", "", "central-db", "external volume name for Central DB")
+	c.Flags().StringVarP(&external.DB.StorageClass, "storage-class", "", "", "storage class name for Central DB (optional if you have a default StorageClass configured)")
+	c.Flags().Uint32VarP(&external.DB.Size, "size", "", 100, "external volume size in Gi for Central DB")
+	return c
+}
+
+func noVolume(cliEnvironment environment.Environment) *cobra.Command {
```

Comment:
Should we keep this command or have this as the default case for `roxctl central db generate`?

Reply:
This is to keep it consistent with the "roxctl central generate" set of commands.